app/controllers/callbacks_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Class handling callbacks from google authentication api
+class CallbacksController < ApplicationController
+  def google_oauth2
+    @user = User.from_omniauth(request.env['omniauth.auth'])
+    # devise method
+    # binding.pry
```

Comment:
this commend is not very helpful, so it should not be in PR

Reply:
Would try to take make function names more meaningful instead of using comments